### DISCUSSION TITLE
please add feature for skipping songs without voting, thanks

### DISCUSSION THREAD

**Discussion by TOP1FORCEY** [jagrosh/MusicBot#1248] [Ideas]
please add feature for skipping songs without voting, so it can be changed in config, very neat feature, thanks

**Comment by jagrosh** [comment]
There are already 3 ways to do this:

1. If you are the person who added the song, using `skip` immediately skips the song without voting.
2. If you are an admin or DJ, the `forceskip` command immediately skips without voting.
3. If you set the skip ratio to 0, anyone using the `skip` command will skip the song without voting.

**Reply by TOP1FORCEY** [reply]
thanks
